test/evergreen.yml

```diff
@@ -2067,6 +2067,35 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: static-wt-build-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-static
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+
+            # -V option displays Wiredtiger library version
+            ./wt -V
+
+            if [ $? -ne 0 ]; then
+              echo "Error, WT util was not generated or is not functioning"
+              exit 1
+            fi
+
+            ldd wt || FAIL=1
```

Comment:
I was a bit confused while reading the FAIL setting and the later checking to determine errors. `FAIL=1` here actually means failing to run `ldd wt` i.e. wt was statically built. Suggest either changing the boolean variable name, or adding a descriptive comment.

Reply:
Fixed